src/plugins/intel_cpu/src/nodes/transpose.cpp

```diff
@@ -37,8 +37,56 @@ bool Transpose::isSupportedOperation(const std::shared_ptr<const ov::Node>& op,
     return true;
 }
 
+class TransposeShapeInfer : public ShapeInferEmptyPads {
+public:
+    TransposeShapeInfer(const size_t& out_rank, const std::vector<size_t>& axes_vec)
+    : m_out_rank(out_rank), m_axes_vec(axes_vec), m_outputShape(std::move(VectorDims(out_rank, 1))), m_needReverse(axes_vec.empty()) {}
```

Comment:
std::move here is not necessary since `VectorDims(out_rank, 1)` is already an rvalue. Moreover you can directly call the constructor `m_outputShape(out_rank, 1)`,

Reply:
> 

correct this bug.